rclpy/services/minimal_client/client.py

```diff
@@ -27,10 +25,7 @@ def main(args=None):
     req = AddTwoInts.Request()
     req.a = 41
     req.b = 1
-    # TODO(mikaelarguedas) remove this once wait for service implemented
-    # wait for connection to be established
-    # (no wait for service in Python yet)
-    time.sleep(1)
+    cli.wait_for_service()
```

Comment:
Now that I've approved it 😄, I noticed that this will block forever. If you look at the C++ version it will do wait_for_service for say a second, then loop with a message like "still waiting for service..."

Not required, just an idea to exercise the timeout and return value.

Reply:
af6a2ef9acd03ef48f1c18f2e60ba8c9827329f3